src/Domain/MigrationStep/s050_DestinationPimInstallation/DestinationPimVersionChecker.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\PimMigration\Domain\MigrationStep\s050_DestinationPimInstallation;
+
+use Akeneo\PimMigration\Domain\Command\ChainedConsole;
+use Akeneo\PimMigration\Domain\Command\SymfonyCommand;
+use Akeneo\PimMigration\Domain\Pim\DestinationPim;
+
+/**
+ * Checks that the destination has all the required end-points
+ *
+ * @author    Laurent Petard <laurent.petard@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ */
+class DestinationPimVersionChecker
+{
+    const EXACT_MAJOR_VERSION = 2;
+    const EXACT_MINOR_VERSION = 0;
+    const MINIMUM_PATCH_VERSION = 3;
+
+    /** @var ChainedConsole */
+    private $console;
+
+    public function __construct(ChainedConsole $console)
+    {
+        $this->console = $console;
+    }
+
+    public function check(DestinationPim $pim)
+    {
+        $systemInformation = $this->console->execute(new SymfonyCommand('pim:system:information'), $pim)->getOutput();
```

Comment:
It's weird to launch command from here. Could be better to create a dedicated service to call in your command and from here.

Reply:
Ideally, yes. But I thought that it wasn't worth it considering that all Transporteo has been built that way. We have to decide a convention to name this type of service and how to use them.

Reply:
That's a recurrent question. It's difficult to know how to call the command from the domain. Anyway it's not part of this PR and as Laurent said Transporteo is built on top of this console system and 3 refactoring stuff have been made on it ^^. It would be cool to rediscuss around it ;),